test/test_helper.rb

```diff
@@ -9,14 +9,6 @@
 require 'phenix'
 require 'pry'
 
-GlobalUid::Base.global_uid_options = {
-  :disabled   => false,
-  :id_servers => [
-    "test_id_server_1",
-    "test_id_server_2"
-  ]
-}
-
```

Comment:
Don’t we need to call `restore_defaults!` here?

Reply:
No, that's being called in the `before` block of the `GlobalUid` test